tests/SimpleStringTest.cpp

```diff
@@ -480,6 +480,13 @@ TEST(SimpleString, Characters)
     CHECK(s == s2);
 }
 
+TEST(SimpleString, CharactersAsDecimals)
+{
+    SimpleString s(DecimalStringFrom(-5));
+    SimpleString s2(DecimalStringFrom(-5));
+    CHECK(s == s2);
```

Comment:
If you test the CHECK, it might be useful. But in this case, you test the DecimalStringFormat, then it isn't useful whatsoever :)